ui/app/components/ui/list-item/index.scss

```diff
@@ -44,11 +44,6 @@
     display: flex;
     align-items: center;
 
-    > h2 {
```

Comment:
We'll still need this for other instances of the list-item, but you can put a specific class on this so you can override the behavior in `asset-list-item`.

Reply:
8a3306f